interface/main/messages/messages.php

```diff
@@ -409,6 +410,35 @@
         $("#new_note").submit();
     }
 });
+ function gotoReport(doc_id,pname,pid,pubpid,str_dob){
+		EncounterDateArray=new Array;
+        CalendarCategoryArray=new Array;
+        EncounterIdArray=new Array;
+        Count = 0;
+	<?php
+		if(isset($enc_list) && sqlNumRows($enc_list) >0 ){
+			while($row = sqlFetchArray($enc_list)){
+	?>
+				        EncounterIdArray[Count]='<?php echo attr($row['encounter']); ?>';
+                                        EncounterDateArray[Count]='<?php echo attr(oeFormatShortDate(date("Y-m-d", strtotime($row['date'])))); ?>';
+                                        CalendarCategoryArray[Count]='<?php echo attr(xl_appt_category($row['pc_catname'])); ?>';
+                                        Count++;
+	<?php
+			}
+		}
+	?>
+	$.ajax({
+		type:'get',
+		url:'<?php echo $GLOBALS['webroot']."/interface/patient_file/encounter/patient_encounter.php";?>',
+		data:{set_pid: pid},
+		async: false
+	});
```

Comment:
TODO for bradymiller
I do not think this ajax call is required. Will remove and test it.